test/runtests.jl

```diff
@@ -23,6 +23,7 @@ end
         rng = StableRNG(157)
         classes = ["class_$i" for i in 1:5]
         c, n = 5, 3
+        Random.seed!(0)
```

Comment:
I wonder if it's possible to set a `StableRNG` as the global RNG...

Reply:
yeah, that would be nice...